javascript/javascript2/week2/movies/movies.js

```diff
@@ -0,0 +1,1729 @@
+    },
+    {
+        title: "After the Sunset",
+        year: 2004,
+        rating: 6.3,
+        votes: 40364,
+        running_times: 5820,
+    },
+    {
+        title: "After.Life",
+        year: 2009,
+        rating: 5.9,
+        votes: 32865,
+        running_times: 6240,
+    },
+    {
+        title: "Aftermath",
+        year: 2017,
+        rating: 5.7,
+        votes: 12202,
+        running_times: 5640,
+    },
+    {
+        title: "Agent Cody Banks",
+        year: 2003,
+        rating: 5.1,
+        votes: 28946,
+        running_times: 6120,
+    },
+    {
+        title: "Agent Cody Banks 2: Destination London",
+        year: 2004,
+        rating: 4.5,
+        votes: 12154,
+        running_times: 6000,
+    },
+    {
+        title: "Agneepath",
+        year: 2012,
+        rating: 7,
+        votes: 19127,
+        running_times: 10440,
+    },
+    {
+        title: "Agora",
+        year: 2009,
+        rating: 7.2,
+        votes: 57484,
+        running_times: 8460,
+    },
+    {
+        title: "Aguirre, der Zorn Gottes",
+        year: 1972,
+        rating: 8,
+        votes: 42543,
+        running_times: 5700,
+    },
+    {
+        title: "Ah fei zing zyun",
+        year: 1990,
+        rating: 7.6,
+        votes: 13683,
+        running_times: 5640,
+    },
+    {
+        title: "Ain't Them Bodies Saints",
+        year: 2013,
+        rating: 6.4,
+        votes: 18005,
+        running_times: 5760,
+    },
+    { title: "Air", year: 2015, rating: 5.1, votes: 10524, running_times: 5700 },
+    {
+        title: "Air America",
+        year: 1990,
+        rating: 5.7,
+        votes: 23513,
+        running_times: 6780,
+    },
+    {
+        title: "Air Bud",
+        year: 1997,
+        rating: 5.1,
+        votes: 14077,
+        running_times: 5880,
+    },
+];
+
+console.log("Only " + movies.length + " movies selected");
+
+// Movies with short titles
+
+const moviesWithShortTitle = movies
+    .filter((movie) => movie.title.length < 7)
+    .map((movie) => movie.title); // gives only the title
+console.log("\nShort titled movies()\n------------");
+console.log(moviesWithShortTitle);
+
+// movies with long titles
+const moviesWithLongTitle = movies
+    .filter((movie) => movie.title.length > 20)
+    .map((movie) => movie.title);
+console.log("\nLong titled movies()\n------------");
+console.log(moviesWithLongTitle);
+
+//Movies produced between 1980 and 1989
+
+let countMoviesProduced = 0;
+const moviesOf1980s = movies.filter(
+    (movie) => movie.year >= 1980 && movie.year <= 1989
+);
+countMoviesProduced++;
+console.log("\nNumber of Movies from 1980s()\n------------");
+console.log(moviesOf1980s.length);
+
+//console.log(moviesOf1980s); // array of movies from 1980s
+
+//Adding an extra tag
+
+movies[5].key = "Tag"; //To add a new key called Tag
+movies[5].values = "Newvalue"; //To add a value pair based on rating
+
+const addingExtraTag = movies.forEach((movie) => {
```

Comment:
This is not okay variable name...try
```suggestion
const moviesWithRating= movies.forEach((movie) => {
```